plugins/EnviroSense.Plugins.PostgresRepositories/Repositories/ApiKeyRepository.cs

```diff
@@ -25,9 +31,9 @@ public async Task<DeviceApiKey> GetByHashAsync(string hash)
         return apiKey;
     }
 
-    public async Task<DeviceApiKey> GetByIdAsync(Guid deviceId)
+    public async Task<ApiKey> GetByIdAsync(Guid Id)
```

Comment:
parameter name should start with lower case, always.


```suggestion
    public async Task<ApiKey> GetByIdAsync(Guid id)
```